content/updates/2021-05/5-API-categories-improvements.md

```diff
@@ -0,0 +1,14 @@
+---
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Easier category trees synchronization within your connectors
+::: meta-data type="Improvement" features="Productivity" available="late May" in="EE,GE" link-to-doc="https://api.akeneo.com/documentation/filter.html#by-category-level-root"
+
+We introduce two new REST API improvements to help you synchronize more efficiently your category trees in your connectors. Indeed, you can now ask to retrieve only tree roots, and you will also find in the category format the date of its last update.
```

Comment:
```suggestion
We introduced two new REST API improvements to help you synchronize more efficiently your category trees in your connectors. Indeed, you can now ask to retrieve only category tree roots, and you will also find its last updated date in the category format.
```

Reply:
It might sound like the last updated date will only apply to category trees. It's true for all categories.

Reply:
```suggestion
We introduced two new REST API improvements to help you synchronize more efficiently your categories and their related trees in your connectors. Indeed, you can now find the last update date directly inside the category format. Also, you can now ask to retrieve only category tree roots thanks to a new query filter.
```